server/app/views/applicant/ProgramIndexTemplate.html

```diff
@@ -82,35 +82,39 @@ <h2 th:text="#{title.getStartedSection}"></h2>
             th:replace="~{applicant/ProgramCardsSectionFragment :: cardsSection(${commonIntakeSection.get()})}"
           ></div>
         </div>
-        <div
+        <form
           th:if="${!categoryOptions.isEmpty()}"
+          id="ns-category-filter-form"
           class="grid-container"
-          id="category-filter-form"
+          hx-get="/programs/hx/filter"
+          hx-target="#not-started-programs"
         >
           <fieldset class="grid-row grid-gap">
             <legend th:text="#{label.programFilters}"></legend>
             <th:block th:each="categoryOption : ${categoryOptions}">
-              <div class="filter-chip">
+              <div>
                 <input
-                  th:id="${'check-category-' + #strings.replace(categoryOption, ' ', '-')}"
+                  th:id="${'ns-check-category-' + #strings.replace(categoryOption, ' ', '-')}"
```

Comment:
(question) Why does this need the `ns-` prefix?

Reply:
I just needed to differentiate it because of the click listener in `main.ts` for the pre-North Star implementation.  I can remove that 'ns' once we've fully moved over to the NS.